src/ChipInput.js

```diff
@@ -295,18 +311,20 @@ class ChipInput extends React.Component {
     } else {
       this.setState({ inputValue: event.target.value })
     }
+    if (this.props.onKeyUp) { this.props.onKeyUp(event) }
   }
 
   handleKeyPress = (event) => {
     this.setState({ keyPressed: true })
+    if (this.props.onKeyPress) { this.props.onKeyPress(event) }
   }
 
   handleUpdateInput = (e) => {
     this.setState({ inputValue: e.target.value })
 
     if (this.props.onUpdateInput) {
       // this.props.onUpdateInput(searchText, dataSource, params)
-      this.props.onUpdateInput(e.target.value)
+      this.props.onUpdateInput(e)
```

Comment:
This is a breaking change. Maybe change it to `this.props.onUpdateInput(e.target.value, e)` to make it non-breaking? :thinking: